src/server/pps/cmds/cmds.go

```diff
@@ -360,6 +367,8 @@ Examples:
 		"this job (accepts job ID)")
 	getLogs.Flags().StringVar(&commaInputs, "inputs", "", "Filter for log lines "+
 		"generated while processing these files (accepts PFS paths or file hashes)")
+	getLogs.Flags().BoolVar(&raw, "raw", false, "Return just the log messages without "+
```

Comment:
I'm not sure what's the most useful actually, if you're getting logs from a particular job and a particular datum then `raw` totally makes sense but if you're getting logs just from a job then it might be handy to see which datum each log message came from.